index_de.md

```diff
@@ -4,52 +4,52 @@ layout: signed
-
-In jüngster Zeit gab verwerfliche Online-Angriffe mit dem Ziel ihn
-aus dem Vorstand der FSF zu entfernen, weil er seine persönlichen
-Meinungen zum Ausdruck gebracht hat. Wir mussten  
-dabei zuschauen wie dies bereits in organisierter Form anderen
-Aktivisten der Freien Software-Bewegung passiert ist.
-Wir werden dieses Mal nicht tatenlos dabei zusehen, wie ein Symbol 
-dieser Gemeinschaft angegriffen wird.
+Richard M. Stallman, häufig bekannt unter dem Kürzel RMS, war seit
+Jahrzehnten eine treibende Kraft in der Freien Software-Bewegung,
+mit Beiträgen wie dem GNU Betriebssystem und Emacs.
+
+In jüngster Zeit wurden eine reihe verwerflicher Online-Angriffe
+bewerkstelligt, mit dem Ziel ihn aus dem Vorstand der FSF zu entfernen,
+weil er seine persönlichen Meinungen zum Ausdruck gebracht hat. Wir
+mussten schon öfter dabei zuschauen, wie dies bereits anderen Aktivisten
+der Freien Software-Bewegung passiert ist. Wir wollen nicht weiter
+tatenlos dabei zusehen, wie ein Symbol dieser Gemeinschaft angegriffen
+wird.
 
 Die FSF ist ein autonomes Gremium, welches durchaus in der Lage ist, 
-seine Mitglieder fair und unvoreingenommen zu behandeln und sollte 
-dem sozialen Druck von außen nicht nachgeben. Wir fordern die 
-FSF auf, die Argumente gegen RMS objektiv zu betrachten
-und die wahre Bedeutung seiner Worte und Taten zu verstehen.
-
-In der Vergangenheit hat RMS seine Ansichten in einer Weise 
-zum Ausdruck gebracht, die viele Menschen verärgerte. Er konzentriert 
-sich in der Regel mehr auf den philosophischen Hintergrund sowie 
-auf das Nacheifern nach Wahrheit und pflegt sprachlichen Purismus. 
-Auf der anderen Seite räumt er den Gefühlen der Menschen in seinen 
-Kommentaren oft zu wenig Raum ein. 
-
-Das macht seine Argumente anfällig für Missverständnisse und 
-Falschdarstellungen, was unserer Meinung nach aus dem offenen Brief, 
-in dem seine Absetzung gefordert wird, klar hervorgeht.
-Seine Worte müssen unter Berücksichtigung, dass er in den meisten Fällen 
-die Dinge nicht diplomatisch zum Ausdruck bringt, interpretiert werden. 
-
-Unabhängig davon: Stallman's Meinungen, für die er verfolgt wird, 
-sind nicht relevant hinsichtlich seiner Fähigkeit, eine Gemeinschaft wie 
-die FSF zu führen. 
-Darüber hinaus hat er genauso Anspruch auf seine eigenen Meinungen 
-wie jeder andere auch. Mitglieder und Unterstützer 
-müssen nicht zwingend mit seinen Ansichten übereinstimmen, 
-sollten aber sein Recht auf Gedanken- und Redefreiheit respektieren.
+seine Mitglieder fair und unvoreingenommen zu behandeln und sollte
+politischem Druck von außen nicht nachgeben. Wir fordern die FSF auf,
+die Argumente gegen RMS objektiv zu betrachten und die eigentliche
+Bedeutung seiner Worte und Taten — im lichte der persönlicher Freiheit,
+die ja den Hauptwert der Freien Software Bewegung darstellt — zu
+verstehen.
+
+In der Vergangenheit hat RMS seine Ansichten in einer Weise zum
+Ausdruck gebracht, die durchaus verärgerten könnte. Er konzentriert 
```

Comment:
Original "... in ways that upset many people." has a different meaning than "durchaus verärgerten könnte".
We shouldn't downplay by using subjunctives.